test/CMakeLists.txt

```diff
@@ -1,14 +1,16 @@
-cmake_policy(SET CMP0135 NEW)
+find_package(GTest)
 
-include(FetchContent)
-include(GoogleTest)
+if(NOT GTest_FOUND)
+    if(POLICY CMP0135)
+        cmake_policy(SET CMP0135 NEW)
+    endif()
 
-set(BUILD_GMOCK OFF)
-set(INSTALL_GTEST OFF)
-set(gtest_force_shared_crt ON)
+    include(FetchContent)
+
```

Comment:
Where is ``include(GoogleTest)``? Isn't it required later?

Reply:
`find_package(GTest)` will include `GoogleTest` as shown in [FindGTest.cmake](https://github.com/Kitware/CMake/blob/df52b85b0795404eec9e076ba4e6df7caaf40490/Modules/FindGTest.cmake#L102).

Reply:
Ah, sure, makes sense to not require it manually.